optuna/integration/allennlp.py

```diff
@@ -141,7 +115,9 @@ def _get_environment_variables_for_pruner() -> Dict[str, Optional[Union[str, int
     kwargs = {}
     for key in keys.split(","):
         key_without_prefix = key.replace("{}_".format(_PREFIX), "")
-        kwargs[key_without_prefix] = _infer_and_cast(os.getenv(key))
+        value = os.getenv(key)
+        assert value is not None
```

Comment:
How about using `ValueError` instead of `assert`? `assert` can be ignored by using `-O`.

Reply:
Thanks. I adopted your suggestion in d4148c5.